lips/lip-3.md

```diff
@@ -7,3 +7,82 @@ type: Informational
 created: 06/09/2020
```

Comment:
This is a great doc.  I would like it to live in the Libra core repository so it can be updated with minor changes (e.g. reformatting/typo fixes, more detail) outside of the LIP process.  It seems like a good fit for Libra specifications (https://github.com/libra/libra/tree/master/specifications), thoughts?